starknet_py/net/schemas/rpc.py

```diff
@@ -105,18 +107,22 @@ def make_dataclass(self, data, **kwargs) -> L2toL1Message:
 class TransactionReceiptSchema(Schema):
     hash = Felt(data_key="transaction_hash", required=True)
     status = StatusField(data_key="status", load_default=None)
+    execution_status = ExecutionStatusField(
+        data_key="execution_status", load_default=None
+    )
+    finality_status = FinalityStatusField(data_key="finality_status", load_default=None)
     block_number = fields.Integer(data_key="block_number", load_default=None)
     block_hash = Felt(data_key="block_hash", load_default=None)
     actual_fee = Felt(data_key="actual_fee", required=True)
     type = TransactionTypeField(data_key="type", load_default=None)
     contract_address = Felt(data_key="contract_address", load_default=None)
-    rejection_reason = fields.String(data_key="status_data", load_default=None)
+    rejection_reason = fields.String(data_key="revert_reason", load_default=None)
```

Comment:
There's no reason whatsoever for the classes to have differently named fields than those in the incoming data.
We should fix that. Here and in other places. Places which are not connected to RPC v4 should be done in #1119.